src/module/scene/token-document/document.ts

```diff
@@ -18,6 +18,9 @@ class TokenDocumentPF2e<TParent extends ScenePF2e | null = ScenePF2e | null> ext
 
     declare auras: Map<string, TokenAura>;
 
+    /** Whether this token is currently moving */
+    isMoving = false;
```

Comment:
Core handles this with `Token#_animation` (worth checking that it is still current in V12)

Reply:
The region test is called from `_preUpdateOperation` so the animation hasn't started yet. That also means any region triggers are fired instantly instead of at the end of the animation. Might be worth bringing up as a core issue.

Reply:
Renamed to `positionChanged` which might be clearer.

Reply:
This is now `newPosition`. The post-update top left Point of the token.